src/code42cli/cmds/profile.py

```diff
@@ -58,32 +62,42 @@ def show(profile_name):
 @name_option
 @server_option
 @username_option
+@password_option
 @disable_ssl_option
-def create(name, server, username, disable_ssl_errors=False):
+def create(name, server, username, password, disable_ssl_errors):
     """Create profile settings. The first profile created will be the default."""
     cliprofile.create_profile(name, server, username, disable_ssl_errors)
-    _prompt_for_allow_password_set(name)
+    if password:
+        _set_pw(name, password)
+    else:
+        _prompt_for_allow_password_set(name)
     echo("Successfully created profile '{}'.".format(name))
 
 
 @profile.command()
 @name_option
 @server_option
 @username_option
+@password_option
 @disable_ssl_option
-def update(name=None, server=None, username=None, disable_ssl_errors=None):
+def update(name, server, username, password, disable_ssl_errors):
     """Update an existing profile."""
     profile = cliprofile.get_profile(name)
     cliprofile.update_profile(profile.name, server, username, disable_ssl_errors)
-    _prompt_for_allow_password_set(profile.name)
+    if password:
```

Comment:
Could be extracted into a private function and shared with lines 70-73

Reply:
this feels like too small of a unit to get any real benefit from pulling it out since it's only used twice and the logic of the code itself is as readable as any function name we could give it.